tests/unit/basic_tests.F90

```diff
@@ -51,7 +51,7 @@ Subroutine test_create(test_id, err_msg)
         ! Error in PIO_createfile
          print *,' ret_val = ', ret_val
         err_msg = "Could not create " // trim(filename)
-        call mpi_abort(MPI_COMM_WORLD, 0, ret_val2)
+        call mpi_abort(MPI_COMM_WORLD, 1, ret_val2)
```

Comment:
Can you add the generic error code as a constant (e.g., integer, parameter :: ERR = 1)?

Reply:
OK, I will update it.